hw1/02-palindrome.js

```diff
@@ -1,3 +1,23 @@
 const elem = document.querySelector('input');
-
+const resultText = document.getElementById('resultText');
 elem.addEventListener('input', handleInput);
+
+function handleInput(){
+    let input = parseInt(elem.value);
```

Comment:
this `input` is not changed later on so I think it's better to use `const` instead